ur_ros2_control_demos/launch/ur5e_run_move_group.launch.py

```diff
@@ -120,11 +172,16 @@ def generate_launch_description():
     ros2_control_node = Node(
         package='controller_manager',
         executable='ros2_control_node',
-        parameters=[robot_description, ur5_controller],
+        parameters=[robot_description, ur_controller],
         output={
             'stdout': 'screen',
             'stderr': 'screen',
         },
     )
 
-    return LaunchDescription([ rviz_node, static_tf, robot_state_publisher, run_move_group_node, mongodb_server_node, ros2_control_node ])
+    if use_ros2_control:
```

Comment:
Can you start it without?

Reply:
It can be started with fake joint driver instead